packages/volto/src/config/Blocks.jsx

```diff
@@ -351,6 +351,12 @@ const blocksConfig = {
     restricted: false,
     mostUsed: true,
     sidebarTab: 1,
+    allowedPeertubeInstances: [
```

Comment:
This setup needs to be documented. Maybe @stevepiercy  can help.

Reply:
I've just added some basic info